### ISSUE TITLE
AzureRmWebDeployment File transform unexpected characters on multi-line comments

### ISSUE THREAD

**Issue by AlexGuirao** [Microsoft/vsts-tasks#5765]
## Troubleshooting
Checkout how to troubleshoot failures and collect debug logs: https://docs.microsoft.com/en-us/vsts/build-release/actions/troubleshooting

## Environment
 VSTS with private Agent
    
## Issue Description

While deploying an app with AzureRmDeployment with file transform activated, we are expiriencing and issue on the transformed web.config files. An un expected character `--&gt;` is appearing after multi line comments in the original config file.

![image](https://user-images.githubusercontent.com/11752100/32371195-401b6ef6-c090-11e7-8ccf-13fb27e42239.png)

After the transformation and deployment, what we have in the actual site is the following.

![image](https://user-images.githubusercontent.com/11752100/32371485-511e398a-c091-11e7-81c8-782991e05add.png)

It started happing as of thi commit [proxy support (#5379)](https://github.com/Microsoft/vsts-tasks/commit/2c87a822d5a61f9212cfa9cb5a93f5a5e52d9eb3) AzureRmDeployment task version 3.3.24

**Comment by vincentdass** [comment]
@AlexGuirao , sorry for the late response.
Currently, we are working on finding the root cause of the issue.

**Comment by vincentdass** [comment]
@AlexGuirao , we identified the root cause and resolved the issue.
We planned for hot-fix which will take a day or two.
In the meantime, you can unblock yourself by removing comments from config files.

**Comment by AlexGuirao** [comment]
@vincentdass Thanks for the information. We removed the multi lines comments in order to keep on working, thanks. 😄

**Comment by frankfuu** [comment]
hi @vincentdass, any ETA on release date? and will this be available for VSTS (cloud)?

**Comment by vincentdass** [comment]
@frankfuu , @AlexGuirao ,
The issue is hot-fixed. Can you please re-run the release and observe the behavior?

**Comment by frankfuu** [comment]
hey @vincentdass,

Still not working for me. Which release is this?  it still using `3.3.24` which is the same number that was causing issues.

```
2017-11-09T07:14:38.8125604Z ##[section]Starting: Deploy Azure App Service
2017-11-09T07:14:38.8395564Z ==============================================================================
2017-11-09T07:14:38.8395564Z Task         : Azure App Service Deploy
2017-11-09T07:14:38.8395564Z Description  : Update Azure WebApp Services On Windows, Web App On Linux with built-in images or docker containers, ASP.NET, .NET Core, PHP, Python or Node based Web applications, Function Apps, Mobile Apps, Api applications, Web Jobs using Web Deploy / Kudu REST APIs
2017-11-09T07:14:38.8395564Z Version      : 3.3.24
2017-11-09T07:14:38.8395564Z Author       : Microsoft Corporation
2017-11-09T07:14:38.8395564Z Help         : [More Information](https://aka.ms/azurermwebdeployreadme)
2017-11-09T07:14:38.8395564Z ==============================================================================
```

**Comment by vincentdass** [comment]
@frankfuu , hotfix is not completed in all the regions.
Will update the thread once it is done.

**Comment by frankfuu** [comment]
hi @vincentdass, this seems to be fixed in `3.3.25` of Azure App Service Deploy.

Thanks!

```
2017-11-10T01:05:31.6765238Z ##[section]Starting: Deploy App Service
2017-11-10T01:05:31.6921483Z ==============================================================================
2017-11-10T01:05:31.6921483Z Task         : Azure App Service Deploy
2017-11-10T01:05:31.6921483Z Description  : Update Azure WebApp Services On Windows, Web App On Linux with built-in images or docker containers, ASP.NET, .NET Core, PHP, Python or Node based Web applications, Function Apps, Mobile Apps, Api applications, Web Jobs using Web Deploy / Kudu REST APIs
2017-11-10T01:05:31.6921483Z Version      : 3.3.25
2017-11-10T01:05:31.6921483Z Author       : Microsoft Corporation
2017-11-10T01:05:31.6921483Z Help         : [More Information](https://aka.ms/azurermwebdeployreadme)
2017-11-10T01:05:31.6921483Z ==============================================================================
2017-11-10T01:05:38.5273382Z XML variable substitution applied successfully.
```
